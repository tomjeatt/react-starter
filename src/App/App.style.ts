import styled, { css } from 'styled-components';

export const Container = styled.div(() => {
  return css`
    background-color: grey;
    margin: 40px auto;
  `;
});
