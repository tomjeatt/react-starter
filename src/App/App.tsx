import React from 'react'
import { Container } from './App.style'

const App: React.FC = () => {
  return (
    <Container>
      <h1>Parcel, React, TypesScript, Styled Components</h1>
    </Container>
  )
}

export default App
