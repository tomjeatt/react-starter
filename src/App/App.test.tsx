import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import App from '.';

afterEach(cleanup);

describe('Application renders', () => {
  it('renders', () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});
