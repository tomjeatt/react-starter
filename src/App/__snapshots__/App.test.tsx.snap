// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application renders renders 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBwQ FmhUY"
  >
    <h1>
      Parcel, React, TypesScript, Styled Components
    </h1>
  </div>
</DocumentFragment>
`;
